src/DETHRACE/common/network.c

```diff
@@ -506,7 +506,14 @@ void NetPlayersChanged(int pNew_count, tNet_game_player_info* pNew_players) {
     LOG_TRACE("(%d, %p)", pNew_count, pNew_players);
 
     if (gCurrent_net_game->type == eNet_game_type_tag || gCurrent_net_game->type == eNet_game_type_foxy) {
-        old_fox_it = gNet_players[gIt_or_fox].ID;
+#ifdef DETHRACE_FIX_BUGS
```

Comment:
the value of `old_fox_it` is undefined at the start of the game because it is accessing `gNet_players[-1].ID` ?

Reply:
In the Carm95 splatpack executable, `gNet_players[-1].ID` maps to the name of the 52th car (max index = 59).
So I think this would break if you install more then 52 cars in your Carmageddon data folder.